### ISSUE TITLE
Configuration to ignore max-line-length for strings

### ISSUE THREAD

**Issue by mklnwt** [pinterest/ktlint#2453]
Add a configuration to allow for long strings exceeding `max-line-length`.
Splitting long lines of strings over multiple lines can obstruct readability of code.

Optional: differentiation between single (`"`) and multiline (`"""`) strings.

```Kotlin
// Assume that the last allowed character is
// at the X character on the right                         X
log.info {
    "Long line of logging you might want to keep in one line"
}
```

## Expected Behavior
Rule should ignore the long string if it's the only code in line or if the line could not be broken down otherwise.

## Current Behavior
> Exceeded max line length (?) (cannot be auto-corrected) (standard:max-line-length)

## Additional information
* Current version of ktlint: 1.1.0

**Comment by paul-dingemans** [comment]
Raw string literals ("""...""") are already excluded from max line length violations as they are typically used in multiline contexts like:
```
log.info {
    """
    Long line of logging you might want to keep in one line
    """
}
```

Strings on the other hand are often used in combination with other elements on the same line. It then becomes unclear what to do if that line exceeds the maximum. Should it be accepted because the line contains a string anywhere, or only when it is the last element on the line? And what if it is the last element, but followed by an EOL comment.

But it might be reasonable to allow lines that only contain on single string literal to exceed the max line length. This is something that can be considered for the next minor release as this is not a regression bug.

**Comment by mklnwt** [comment]
Last element on the line is the most important for now (i.e. my logging example).

A bit trickier would be:
```Kotlin
log.info {
    "Long line of logging with a variable ${foo.bar} you might want to keep in one line"
}
```

I would want to still keep that in one line.

Let me think a little deeper about other cases.

**Comment by paul-dingemans** [comment]
> log.info {
>     "Long line of logging with a variable ${foo.bar} you might want to keep in one line"
> }

From the parser perspective this log statement still contains a single string. Following would however not be accepted:
```
log.info {
    "Long line of logging with a variable " + foo.bar + " you might want to keep in one line"
}```
